src/main/java/microsoft/exchange/webservices/data/AttachmentCollection.java

```diff
@@ -352,7 +352,7 @@ public void validate() throws Exception {
     for (int attachmentIndex = 0; attachmentIndex < this.getAddedItems()
         .size(); attachmentIndex++) {
       Attachment attachment = this.getAddedItems().get(attachmentIndex);
```

Comment:
since the `Attachment` will not be changed again perhabs it should be declared as final. Same with the `FileAttachment` on line 376